packages/cli/test/simulation/singleNodeForks.test.ts

```diff
@@ -12,6 +12,7 @@ describe("singleNodeForks", function () {
       validatorsPerClient: 128,
       altairEpoch: 1,
       bellatrixEpoch: 2,
+      logFilesDir: `${logFilesDir}/singleNodeForks`,
```

Comment:
Test identification should be much more specific. Please check existing sim tests

Reply:
I mapped the test files name with the log directory name. This is a lot easier approach to identify logs for a test file. 

In earlier tests, the processes are created multiple times in one test file, so why have a prefix for configuration.